libs/models/src/lib/sockets/player.ts

```diff
@@ -42,6 +42,7 @@ export interface Player {
 }
 
 export interface PlayerWithLogs extends Omit<Player, 'state' | 'lobbyId'> {
+  //rem lobbyId
```

Comment:
What's this?

Reply:
Removed unnecessary comments. 755e6524b5cc463844463d50060be3604003a6a2